src/schematics/library-versions.ts

```diff
@@ -0,0 +1 @@
+export const singleSpaAngular = '^2.1.0';
```

Comment:
Should we change this to maybe `^3.0.0-beta.1`? And then I'll publish this as 3.0.0-beta.1 and the correct version will be added to people's package.json?

Reply:
I'll merge this in and make the change myself real quick. Then publish a beta.